### PR TITLE
New package: colord-kde-0.5.0

### PR DESCRIPTION
**Pull request by Pricly-Yellow**

### CODE REVIEW

File: srcpkgs/colord-kde/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+depends="colord $(vopt_if gcm gnome-color-manager)"
+short_desc="KDE colord integration"
+maintainer="Danila Kryukov <danila.kryukov.dev@yandex.ru>"
+license="GPL-2.0-or-later"
+homepage="https://cgit.kde.org/colord-kde.git/"
+distfiles="${KDE_SITE}/colord-kde/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=6c545009af737ad67900e4d8549086bc65b979c799b2402871bba44addb52d2f
+
+build_options="gcm"
+desc_option_gcm="Enable support for gnome callibration tools"
```
Comment (Chocimier) [inline srcpkgs/colord-kde/template:18]: If this is just for runtime dependency, there is no need for build option. Leave it for user to find out what's needed, or _maybe_ install note with vdoc.

File: srcpkgs/colord-kde/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="kcoreaddons extra-cmake-modules gettext pkg-config qt5-devel qt5-qmake"
+makedepends="kcmutils-devel libXrandr-devel lcms2-devel plasma-framework-devel"
+depends="colord $(vopt_if gcm gnome-color-manager)"
+short_desc="KDE colord integration"
+maintainer="Danila Kryukov <danila.kryukov.dev@yandex.ru>"
+license="GPL-2.0-or-later"
+homepage="https://cgit.kde.org/colord-kde.git/"
```
Comment (Chocimier) [inline srcpkgs/colord-kde/template:13]: cgit is closed, current address is https://invent.kde.org/graphics/colord-kde

File: srcpkgs/colord-kde/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'colord-kde'
+pkgname=colord-kde
+version=0.5.0
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```
Comment (Chocimier) [inline srcpkgs/colord-kde/template:6]: No need, there are no tests anyway, and if there were we want it.

**Comment by Pricly-Yellow** [comment]
@Chocimier thank you for review. Fix template

File: srcpkgs/colord-kde/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+makedepends="kcmutils-devel libXrandr-devel lcms2-devel plasma-framework-devel"
+depends="colord"
+short_desc="KDE colord integration"
+maintainer="Danila Kryukov <danila.kryukov.dev@yandex.ru>"
+license="GPL-2.0-or-later"
+homepage="https://invent.kde.org/graphics/colord-kde"
+distfiles="${KDE_SITE}/colord-kde/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=6c545009af737ad67900e4d8549086bc65b979c799b2402871bba44addb52d2f
+
+if [ "$CROSS_BUILD" ]; then
```
Comment (Chocimier) [inline srcpkgs/colord-kde/template:16]: Wasn't it building fine without?

Comment (Pricly-Yellow) [reply srcpkgs/colord-kde/template]: Remove this if, build correctly, but in time when CI build my package qt5 updating to 5.15 and CI build qt5 not my pkg. how to restart Travis check?

Comment (ericonr) [reply srcpkgs/colord-kde/template]: You can just rebase the commits on upstream and force-push them.

**Pull request merged**